apps/server/src/modules/user-login-migration/service/school-migration.service.spec.ts

```diff
@@ -250,4 +250,48 @@ describe('SchoolMigrationService', () => {
 			);
 		});
 	});
+
+	describe('restartMigration is called', () => {
```

Comment:
Also please use our test strcture.

Reply:
What is wrong with this? 
Can you please link the documentation (and i mean no example spec) ? ..We talked about a brown bag session in the past for presenting this structure to all project members :) When will it take place?

Reply:
The documentation from last year: https://hpi-schul-cloud.github.io/schulcloud-server/additional-documentation/nestjs-application/testing.html

Maybe more detailed examples: https://docs.dbildungscloud.de/display/DBH/Code+Styles+08.02.23#CodeStyles08.02.23-Teststructure